doc/install/mne_python.rst

```diff
@@ -155,23 +57,19 @@ your operating system.
 
 .. collapse:: |apple| macOS
 
-   Update the ``base`` conda environment to include the ``nb_conda_kernels``
+   Update the "base" conda environment to include the ``nb_conda_kernels``
    package, so you can use MNE-Python in Jupyter Notebooks launched from the
-   Anaconda GUI. Then download the MNE-Python `environment file`_ (done here
-   with ``curl``) and use it to create a new environment (named ``mne`` by
-   default)::
+   Anaconda GUI. Then install MNE-Python into a new environment (here called
+   ``mne``, but you can name the environment whatever you want)::
```

Comment:
This is potentially confusing (note that I'm not an Anaconda user though). Why do I need to install `nb_conda_kernel` in "base" when MNE-Python is in a separate environment? If I start a Jupyter notebook, how do I switch to the `mne` environment then? Maybe this should be briefly mentioned.

Reply:
if you use the anaconda navigator, you get a window of tiles for different apps (Jupyter, vscode, orange, rstudio, terminal...).  There is also a small dropdown that is easy to miss that says which environment you're in (and it defaults to "base" when you launch anaconda navigator).  Even if you know about how to switch envs, it's easy to forget because the UI doesn't offer good affordances for it, and doing it through the GUI is rather slow as well.

If you install `nb_conda_kernels` into base env, then you can launch Jupyter from base, but create and run notebooks who run their kernel through the MNE environment.  See the dropdown box for new notebook creation:

![Screenshot_2021-08-04_09-27-28](https://user-images.githubusercontent.com/1810515/128199838-4f62fbbb-93dc-4f0c-a280-19581b9b03d2.png)

Honestly I think this is a case where the explanation is too long and convoluted to be worth including.  I've reworded it to say 

> If you like using Jupyter notebooks, you should also update the "base" conda environment to include the ``nb_conda_kernels`` package; this will make it easier to use MNE-Python in Jupyter Notebooks launched from the Anaconda GUI.

...followed by the relevant code snippet. OK?

Reply:
Absolutely! Perfect!